app/components/trusted-companies.tsx

```
@@ -0,0 +1,172 @@
+        </div>
+
+        <div className="block sm:hidden">
+          <div className="grid grid-cols-2 gap-6 sm:gap-8">
+            {companies.slice(0, 6).map((company) => (
+              <div
+                key={company.id}
+                className="flex items-center justify-center p-4 sm:p-6 bg-white/5 backdrop-blur-sm rounded-xl border border-white/10 hover:bg-white/10 transition-all duration-300 group"
+              >
+                {company.logo.startsWith('<svg') ? (
+                  <div
+                    dangerouslySetInnerHTML={{ __html: company.logo }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)